src/main/java/com/snowflake/kafka/connector/internal/SnowflakeIngestionServiceV1.java

```diff
@@ -261,9 +261,10 @@ public SimpleIngestManager getIngestManager() {
   /**
    * configure the Snowpipe client and return the client sequencer
    *
-   * @return a Long value contains the client sequencer
+   * @return ConfigureClientResponse contains the client sequencer
    */
```

Comment:
May be you can remove this javadocs from here since we already have it in interface?